internal/providers/terraform/parser_test.go

```diff
@@ -0,0 +1,300 @@
+package terraform
```

Comment:
We already have `parser_internal_test.go`, I think this is left from when we had integration tests for the parser. Do you think we should move all the tests from there into this `parser_test.go` file as well?

Reply:
I think we should, I don't see any reasons to split that tests to two files. Should I move them?

Reply:
Yes that would be great, thanks.